app/models/book_suggestions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class BookSuggestions < ApplicationRecord
```

Comment:
Remember that rails convention for models is singular, this should be `BookSuggestion`